test/e2e/storage/subpath.go

```diff
@@ -116,4 +120,56 @@ var _ = utils.SIGDescribe("Subpath", func() {
 			testsuites.TestBasicSubpath(f, "configmap-value", pod)
 		})
 	})
+
+	Context("NFS volumes", func() {
+		var (
+			nfsServerPod *v1.Pod
+			nfsServerIP  string
+		)
+
+		BeforeEach(func() {
+			_, nfsServerPod, nfsServerIP = framework.NewNFSServer(f.ClientSet, f.Namespace.Name, []string{"-G", "777", "/exports"})
+
+		})
+
+		AfterEach(func() {
+			framework.DeletePodWithWait(f, f.ClientSet, nfsServerPod)
+		})
+
+		It("should be able to unmount a stale NFS handle", func() {
```

Comment:
Other than the nfs driver setup, I think this test case actually can work across any volume type (remove the subpath directory while it's still bind mounted to the container).  Will look into moving this into the framework

Reply:
done